### PR TITLE
Gh pages

### PR DESCRIPTION
**Pull request by huskerman34**
Could you please let me know what im doing wrong? Its way off some where

### CODE REVIEW

**Comment by huskerman34** [comment]
Case study turn in.

**Comment by mbMosman** [comment]
Make sure to get started on these case studies early so that you have time to get help early.  I'm generally not available over the weekends, look for my office hours or make sure you're allowing 24 hrs for email.

File: pacifictrails.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+#contact {
+    font-size: 90%;
+    letter-spacing: normal;
+}
+ul {
+    list-style-image: url(images/marker.gif);
+}
+main {
+    background-color: #ffffff;
+    margin-left: 190px;
```
Comment (mbMosman) [inline pacifictrails.css]: This is not how the padding for the main element is specified in the text-book.  Look back through each chapter to get this set up correctly.

File: pacifictrails.css [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    height: 300px;
+    background-size: 100% 100%;
+    background-repeat: no-repeat;
+}
+    nav ul {
+    list-style-image: none;
+    list-style-type: none;
+    margin: 0;
+    padding-left: 0;
+    font-size: 1.0em;
```
Comment (mbMosman) [inline pacifictrails.css]: The font size for the nav should be larger.

File: yurts.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                <li> <a href="index.html"> Home</a> &nbsp;</li>
+                <li> <a href="yurts.html"> Yurts</a> &nbsp;</li>
+                <li> <a href="Activities.html"> Activities </a> &nbsp;</li>
+                <li><a href="Reservation.html">  Reservations</a> &nbsp;</li>
+            </ul>
+        </nav>
+        <main>
+            <div id="yurthero"></div>
+            <h2>The Yurts at Pacific Trails</h2>
+            <dl>
```
Comment (mbMosman) [inline yurts.html]: This should be set up like the activities page with h3 and paragraph elements not a dl with dt/dd tags.

File: yurts.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+                <dt><strong> What is a Yurt?</strong></dt>
+                <dd>Our luxury yurts are permenant structures four feet off the ground. Each yurt has canvas walls, a wooden floor, and a roof dome that can be opened.</dd>
+                <dt><strong>How are the yurts furnished?</strong></dt>
+                <dd> Each yurt is furnished wiht a queen-size bed with down quilt and gas-fired stove. The luxury camping experience also includes electricity and sink with hot and cold running water.Shower and restroom facilities are located in the lodge.</dd>
+                <dt><strong>What should i bring?</strong></dt>
```
Comment (mbMosman) [inline yurts.html]: I should be capitalized.

File: activities.html [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    <div id="wrapper">
+
+
+    <header>
+        <h1>Pacific Trails Resort</h1></header>
+    <nav>
+        <ul>
+            <li> <a href="index.html"> Home</a> &nbsp;</li>
+            <li> <a href="yurts.html"> Yurts</a> &nbsp;</li>
+            <li> <a href="Activities.html"> Activities </a> &nbsp;</li>
```
Comment (mbMosman) [inline activities.html]: All the links to your activities page are broken on the web due to case-sensitivity.  The file name has a lower-case letter.
Make sure you are testing your page from GitHub on the web.  https://huskerman34.github.io/cs-pacific-trails/activities.html

**Pull request closed without merging**